lib/src/presentation/widgets/error_screen.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:lottie/lottie.dart';
+
+import '../../core/utils/assets_constants.dart';
+import '../../core/utils/dimensions_constants.dart';
+import '../../core/utils/string_constants.dart';
+
+Widget errorWidget(
```

Comment:
Rename the file, you are creating a widget not a screen

Reply:
Done